sdk/resourcemanager/azure-resourcemanager-redis/pom.xml

```diff
@@ -161,6 +161,7 @@
           <artifactId>revapi-maven-plugin</artifactId>
           <version>0.11.2</version> <!-- {x-version-update;org.revapi:revapi-maven-plugin;external_dependency} -->
           <configuration>
+            <skip>true</skip>
```

Comment:
Breaking changes are added into CHANGELOG, below is breaking change error logs:

[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.revapi:revapi-maven-plugin:0.11.2:check (default) on project azure-resourcemanager-redis: The following API problems caused the build to fail:
[ERROR] java.class.nowFinal: class com.azure.resourcemanager.redis.models.RedisCreateParameters: Class is now final. (breaks semantic versioning)
[ERROR] java.annotation.removed: class com.azure.resourcemanager.redis.models.RedisCreateParameters: Element no longer annotated with 'com.azure.core.annotation.JsonFlatten'. (breaks semantic versioning)
[ERROR] java.class.removed: class com.azure.resourcemanager.redis.models.RedisCreateProperties: Class was removed. (breaks semantic versioning)
[ERROR] java.class.nowFinal: class com.azure.resourcemanager.redis.models.RedisFirewallRuleCreateParameters: Class is now final. (breaks semantic versioning)
[ERROR] java.annotation.removed: class com.azure.resourcemanager.redis.models.RedisFirewallRuleCreateParameters: Element no longer annotated with 'com.azure.core.annotation.JsonFlatten'. (breaks semantic versioning)
[ERROR] java.class.nowFinal: class com.azure.resourcemanager.redis.models.RedisLinkedServerCreateParameters: Class is now final. (breaks semantic versioning)
[ERROR] java.annotation.removed: class com.azure.resourcemanager.redis.models.RedisLinkedServerCreateParameters: Element no longer annotated with 'com.azure.core.annotation.JsonFlatten'. (breaks semantic versioning)
[ERROR] java.class.removed: class com.azure.resourcemanager.redis.models.RedisLinkedServerCreateProperties: Class was removed. (breaks semantic versioning)
[ERROR] java.class.removed: class com.azure.resourcemanager.redis.models.RedisLinkedServerProperties: Class was removed. (breaks semantic versioning)
[ERROR] java.class.nowFinal: class com.azure.resourcemanager.redis.models.RedisUpdateParameters: Class is now final. (breaks semantic versioning)
[ERROR] java.annotation.removed: class com.azure.resourcemanager.redis.models.RedisUpdateParameters: Element no longer annotated with 'com.azure.core.annotation.JsonFlatten'. (breaks semantic versioning)
[ERROR] java.class.removed: class com.azure.resourcemanager.redis.models.RedisUpdateProperties: Class was removed. (breaks semantic versioning)
[ERROR] java.class.removed: class com.azure.resourcemanager.redis.models.UpgradeNotification: Class was removed. (breaks semantic versioning)